scripts/console/2023.2.0/001_use_tags.py

```diff
@@ -0,0 +1,57 @@
+from collections import defaultdict
+
+DRY_RUN = True
+
+mef_eline = controller.napps[("kytos", "mef_eline")]
+evcs = {
+    evc_id: evc.as_dict()
+    for evc_id, evc in mef_eline.circuits.items()
+    if not evc.archived
+}
+
+in_use_tags = defaultdict(list)
+for evc_id, evc in evcs.items():
+    for link in evc["current_path"]:
+        svlan = link["metadata"]["s_vlan"]["value"]
+        intfa = link["endpoint_a"]["id"]
+        intfb = link["endpoint_b"]["id"]
+        in_use_tags[intfa].append((svlan, evc_id))
+        in_use_tags[intfb].append((svlan, evc_id))
+    for link in evc["failover_path"]:
+        svlan = link["metadata"]["s_vlan"]["value"]
+        intfa = link["endpoint_a"]["id"]
+        intfb = link["endpoint_b"]["id"]
+        in_use_tags[intfa].append((svlan, evc_id))
+        in_use_tags[intfb].append((svlan, evc_id))
+
+    for uni in ("uni_a", "uni_z"):
+        intf_id = evc[uni]["interface_id"]
+        if (
+            "tag" in evc[uni]
```

Comment:
I just caught an issue on this part that I tried to contribute, for EPLs in since it's reading directly from `mef_eline.circuits`, `tag` can be None too, we'll need this extra safe guard, can you apply these patches below on both console scripts?

```
     27 for uni in ("uni_a", "uni_z"):
     28     intf_id = evc[uni]["interface_id"]
     29     if (
     30         "tag" in evc[uni]
---> 31         and "tag_type" in evc[uni]["tag"]
     32         and evc[uni]["tag"]["tag_type"] in ("vlan", 1)
     33     ):
     34         tag = evc[uni]["tag"]["value"]
     35         if isinstance(tag, int):

TypeError: argument of type 'NoneType' is not iterable
```

```
diff --git a/scripts/console/2023.2.0/001_use_tags.py b/scripts/console/2023.2.0/001_use_tags.py
index 4639be9..6e8c534 100644
--- a/scripts/console/2023.2.0/001_use_tags.py
+++ b/scripts/console/2023.2.0/001_use_tags.py
@@ -28,6 +28,7 @@ for evc_id, evc in evcs.items():
         intf_id = evc[uni]["interface_id"]
         if (
             "tag" in evc[uni]
+            and evc[uni]["tag"]
             and "tag_type" in evc[uni]["tag"]
             and evc[uni]["tag"]["tag_type"] in ("vlan", 1)
         ):
diff --git a/scripts/console/2023.2.0/002_recover_vlans.py b/scripts/console/2023.2.0/002_recover_vlans.py
index db47bce..eda2072 100644
--- a/scripts/console/2023.2.0/002_recover_vlans.py
+++ b/scripts/console/2023.2.0/002_recover_vlans.py
@@ -62,6 +62,7 @@ for evc_id, evc in evcs.items():
         intf_id = evc[uni]["interface_id"]
         if (
             "tag" in evc[uni]
+            and evc[uni]["tag"]
             and "tag_type" in evc[uni]["tag"]
             and evc[uni]["tag"]["tag_type"] in ("vlan", 1)
         ):
```